### PR TITLE
A few updates to the next demo configs

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

**Comment by vishh** [comment]
cc @yguo0905

File: demo/tpu-training/inception-v3-tpu.yaml [hunk 1]
```diff
@@ -64,7 +64,7 @@ spec:
           # [REQUIRED] Must specify the Google Cloud Storage location where your
           # output model will be stored.
         - name: MODEL_BUCKET
-          value: "gs://<my-model-bucket>/inception_v3"
+          value: "gs://vishh/tf-models/inception_v3"
```
Comment (yguo0905) [inline demo/tpu-training/inception-v3-tpu.yaml]: OK to leave your bucket here?

File: demo/serving/k8s_tensorflow_serving.yaml [hunk 2]
```diff
@@ -36,10 +36,26 @@ spec:
         args:
           - --port=9000
           - --model_name=inception
-          - --model_base_path=gs://kubeflow-models/inception/
+          - --model_base_path=gs://vishh/tensorflow/inception/
         resources:
           limits:
             nvidia.com/gpu: 1
+        readinessProbe:
```
Comment (yguo0905) [inline demo/serving/k8s_tensorflow_serving.yaml]: Can you please explain why the probe is important?

Comment (yguo0905) [reply demo/serving/k8s_tensorflow_serving.yaml]: Nvm. I was thinking the other probe.

File: demo/tpu-training/inception-v3-tpu.yaml (hunk 1 above)
Comment (vishh) [reply demo/tpu-training/inception-v3-tpu.yaml]: Yeah. It will be easier for the demo. I'll clean it up later.

**Pull request merged**